stubs/CoreGenericClasses.phpstub

```diff
@@ -20,7 +20,7 @@ interface Traversable {
  */
 class Generator implements Traversable {
     /**
-     * @return TValue Can return any type.
+     * @return ?TValue Can return any type.
```

Comment:
I would probably need [`@psalm-ignore-nullable-return`](https://psalm.dev/docs/annotating_code/supported_annotations/#psalm-ignore-nullable-return) as well. Otherwise code like this will fail:
```php
/** @param Generator<int, int> */
function f(Generator $g): void {
   while ($generator->valid()) {
       takesInt($generator->current());
       $generator->next();
   }
}
```

Reply:
Yup, good suggestion!

Reply:
Done! :)